tests/files/mpeg.rs

```diff
@@ -306,3 +307,57 @@ fn remove_id3v1() {
 fn remove_ape() {
 	crate::remove_tag!("tests/files/assets/minimal/full_test.mp3", TagType::Ape);
 }
+
+#[test]
+fn read_and_write_tpil_frame() {
+	let key_value_pairs = vec![
+					("engineer".to_string(), "testperson".to_string()), 
+					("vocalist".to_string(), "testhuman".to_string())
+				];
+
+	let mut file = temp_file!("tests/files/assets/issue_213.mp3");
+
+	let tagged_file = Probe::new(&mut file)
```

Comment:
Since this isn't a probe test, I'd just use `MpegFile::read_from` so that you can just use `Id3v2Tag` directly.